guides-for-using-cinchy/builder-guides/creating-tables/columns.md

```diff
@@ -194,47 +197,64 @@ Cinchy has several Date column type display format options available:
 {% endhint %}
 
-_These are equivalent to `DATE()` data type in SQL._
-
+{% hint style="info" %}
+These are equivalent to `DATE()` data type in SQL.
+{% endhint %}
 ### Yes/No
 
 You must select a default value of yes (1) or no (0) for yes/no fields.
 
-_These are equivalent to `bit`data type in SQL._
+{% hint style="info" %}
+These are equivalent to `bit`data type in SQL.
+{% endhint %}
+
+### Choice
 
-### Calculated
+You can create a choice column (single or multi-select) in Cinchy. In this scenario, you specify all your choices (1 per newline) in the table design. A user will only be able to select from the options provided.
+## Calculated Columns 
 
-A calculated column is evaluated using other fields on the record. It will also have a result type - which is the form in which the calculated results will be stored.
+A calculated column uses values from other fields in the record for its evaluation. These columns also have a specified result type, which dictates the format of the calculated output.
 
-For example, you can have a column `[Full Name]` that's `CONCAT([First Name], ' ', [Last Name])`_._
+**Example**:  
 
-_These are equivalent to computed columns in SQL._
+A `[Full Name]` column can be calculated as `CONCAT([First Name], ' ', [Last Name])`.
 
-#### Cached vs Live Calculated Columns
+{% hint style="info" %}
+These columns are similar to computed columns in SQL databases.
+{% endhint %}
+### Live vs Cached Calculated Columns
 
-When creating a calculated column, you will notice the option to have it cached or not, located under Advanced Settings. This was an option introduced in version 4.0 of the platform.
+#### Choose Your Calculation Type
 
-![](<../../../.gitbook/assets/image (304).png>)
+When creating a calculated column, you have two types to choose from: **cached** and **live**. This feature is accessible via the Advanced Settings and was a part of the 4.0 version update.
 
-A **cached calculated column** stores your data for fast retrieval and querying. Calculated columns are defaulted to cached. This is an actual column in your table that's based on a defined CQL formula. This column will recalculate when data is changed **in the same row.**
+#### Cached Calculated Columns
 
-In the below example, the **Label** column is a calculated column that connects various name columns together. You can see that _Connect your data_ appears in each label. If you then wanted to change the name in row one from _Connect your data_ to _Connect **all** your data_, only that specific row would recalculate automatically to update the label. To update any other row (within this table or another) with a calculated column that references that data, you would need to manually make changes to each to prompt a recalculation.
+- **What It Does**: Speeds up data retrieval.
+- **How It's Stored**: As an actual column based on a CQL formula.
+- **When It Updates**: Updates only if the data in the same row changes.
```

Comment:
One gotcha is if it's relying on a column from another table and the value in the other table changes, this cached CC will not get updated. I think we should call this out.

Reply:
Adding additional information.

Reply:
Added on line 246